test/02_test/cibuildwheel_test_test.py

```diff
@@ -1,55 +1,61 @@
-import os, subprocess
+import subprocess
+import os
+import sys 
+
 import pytest
-import utils
 
-def test():
-    project_dir = os.path.dirname(__file__)
+sys.path.append(os.path.dirname(os.path.dirname(__file__)))
```

Comment:
We cannot have to repeat this for every test. This is not simplifying things. If this is the only way, I prefer to keep using `run_tests.py`